### PR TITLE
Internal: Add border-radius prop type + transformer [EDS-533]

### PR DESCRIPTION
**Pull request by ronros-elementor**

### CODE REVIEW

File: modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+use Elementor\Modules\AtomicWidgets\PropsResolver\Multi_Props;
+use Elementor\Modules\AtomicWidgets\PropsResolver\Transformer_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Border_Radius_Transformer extends Transformer_Base {
+	public function transform( $value, $key ) {
+		$dimensions = Collection::make( $value )
```
Comment (Omerisra6) [inline modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php:15]:
```suggestion
		$corners = Collection::make( $value )
```

Comment (Omerisra6) [inline modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php:15]:
```suggestion
		$corners = Collection::make( $value )
```

File: modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+use Elementor\Core\Utils\Collection;
+use Elementor\Modules\AtomicWidgets\PropsResolver\Multi_Props;
+use Elementor\Modules\AtomicWidgets\PropsResolver\Transformer_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Border_Radius_Transformer extends Transformer_Base {
```
Comment (Omerisra6) [inline modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php:13]: add test in the style renderer

File: modules/atomic-widgets/prop-types/border-radius-prop-type.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace Elementor\Modules\AtomicWidgets\PropTypes;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Base\Object_Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Border_Radius_Prop_Type extends Object_Prop_Type {
```
Comment (Omerisra6) [inline modules/atomic-widgets/prop-types/border-radius-prop-type.php:11]: add test in the base class: get data for save

**Review by Omerisra6 — CHANGES_REQUESTED** [review]

File: modules/atomic-widgets/prop-types/border-radius-prop-type.php (hunk 3 above)
Comment (ronros-elementor) [reply modules/atomic-widgets/prop-types/border-radius-prop-type.php:11]: Done

File: modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php (hunk 2 above)
Comment (ronros-elementor) [reply modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php:13]: Done

File: modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php (hunk 1 above)
Comment (Omerisra6) [reply modules/atomic-widgets/props-resolver/transformers/styles/border-radius-transformer.php:15]: I am just not sure if the corners are dimensions.

File: modules/atomic-widgets/prop-types/border-radius-prop-type.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	public static function get_key(): string {
+		return 'border-radius';
+	}
+
+	protected function define_shape(): array {
+		return [
+			'top-left' => Size_Prop_Type::make(),
```
Comment (Omerisra6) [inline modules/atomic-widgets/prop-types/border-radius-prop-type.php:18]: does all the props are required?

**Review by Omerisra6 — APPROVED** [review]

**Pull request merged**